Zend/tests/nullsafe_operator/004.phpt

```diff
@@ -8,4 +8,4 @@ var_dump($foo?->bar = 'bar');
 
 ?>
 --EXPECTF--
-Fatal error: Can't use nullsafe operator in write context in %s.php on line 4
+Fatal error: Cannot use the null-safe operator in write context in %s on line %d
```

Comment:
I believe `nullsafe` is not the most correct form. At least, my built-in type checkers are always complaining. I'm not sure if `null safe` or `null-safe` is the best choice, though.

Reply:
``null-safe`` would be the more grammatically correct version from my understanding as it is compound modifying the noun ``operator`` and therefore requires a hyphen according to various sources (https://www.grammarly.com/blog/hyphen/ https://www.gsbe.co.uk/grammar-the-hyphen.html https://dictionary.cambridge.org/grammar/british-grammar/hyphens)

Reply:
Thanks for the background info and the resources! These are always welcome :)